index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
 
   contentFor: function(type) {
     if (type === 'vendor-suffix') {
-      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(__dirname, 'vendor/jquery-waypoints/waypoints.js')) + '\n}';
+      return 'if (typeof(document) !== "undefined") {\n' + fs.readFileSync(path.join(__dirname, '../../bower_components/jquery-waypoints/waypoints.js')) + '\n}';
```

Comment:
Ok, this works starting from a clean project.

Note, that I couldn't find the property to get the bower folder, apparently `this.bowerDirectory` is undefined. It appears that property hangs off of the `app` that is passed into the `included:` hook. So I hard coded it. I need to add the `../..` otherwise I got the error below.

![screen shot 2015-04-23 at 10 10 19 am](https://cloud.githubusercontent.com/assets/61075/7299104/f12cbd26-e9a1-11e4-8ab5-d60b7b1aac20.png)

If you know of a way I can access the application's bower folder without hard coding it, that'd be much appreciated.